pkg/ebpfev_v1/event_interface.go

```diff
@@ -0,0 +1,14 @@
+package ebpfev_v1
```

Comment:
[Question]: What does `ebpfev` stand for?

Reply:
ebpf event